src/components/mobile/PasswordModal.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <MobilePage
+    hide-tab-bar
+    right-button-icon-name="close"
+    @right-button-click="cancelHandler"
+  >
+    <Guide>
+      Enter your password to continue
+    </Guide>
+
+    <form
+      :id="_uid"
+      @submit.prevent="continueHandler"
+    >
+      <PasswordPurpose />
+      <AeInputPassword
+        v-model="password"
+        v-validate="'required|min:4'"
+        autofocus
+        autocomplete="current-password"
```

Comment:
If our goal is to check that user remembers the password, should we autocomplete it?

Reply:
This is a hint for a web browser. Maybe the user has some software/hardware that handles user's passwords in some secure way. I don't think that we should be against it.